pontoon/base/models.py

```diff
@@ -593,6 +615,10 @@ def create_users_map(qs, keyfield):
 
         return locale_projects
 
+    @property
+    def contributors(self, start_date=None):
```

Comment:
I think you won't be able to pass start_date. You could do somethin like this: http://stackoverflow.com/questions/5715620/python-how-to-pass-more-than-one-argument-to-the-property-getter and create a separate method called `get_contributors_since` and create a property that uses specific call of this method.